### ISSUE TITLE
RadListView itemTap & ItemDeselected events order inconsistency on iOS and Android

### ISSUE THREAD

**Issue by NickIliev** [telerik/nativescript-ui-feedback#416]
The order in which `itemTap` along with `ItemSelected` (or `ItemDeselected`) events are raised is different on iOS and on Android (when we have `selectionBehavior` set to `Press`).

For example on Android
```
JS: onItemTap
JS: onItemDeselected
```

On iOS
```
JS: onItemDeselected
JS: onItemTap
```

Reported via t.1141251

Project to reproduce the issue can be found [here](https://github.com/NickIliev/NS-Issues/tree/master/others/t_1141251)

Expected result: investigate if we can align the event raise order for both platforms.

**Comment by felix-idf** [comment]
Are there any updates on when this issue is expected to be fixed?

We would like to simplify the selection/deselection behaviour of all our RadListViews by activating the selection mode on longpress and select/deselect further items using a short tap.

I found a workaround for this by putting the code from itemDeselected-event into the itemtap-event, but I don't like this, as it creates inconsistent code.

**Comment by zbranzov** [comment]
The fix is released in nativescript-ui-listview@3.8.0. Now on both OSs ontItemTap event is raised first.
